HipercowApiUnitTests/Controllers/AuthControllerUnitTest.cs

```diff
@@ -7,16 +7,15 @@ namespace HipercowApiUnitTests.Controllers
     using System.Net;
     using System.Security.Claims;
     using Hipercow_api.Tools.Exceptions;
-    using HipercowApi.Models;
     using HipercowApi.Tools;
     using Microsoft.AspNetCore.Http;
     using Microsoft.AspNetCore.Mvc;
-    using Microsoft.Extensions.Caching.Memory;
     using Moq;
 
     /// <summary>
     /// Test the /auth endpoints.
     /// </summary>
+    [ExcludeFromCodeCoverage]
```

Comment:
Coverage shouldn't be hitting the unit tests folder...?

Reply:
Yeah - I haven't yet found a way from stopping VS from including code coverage of the test project, which is quite annoying. I wouldn't mind so much if it was accurate, but it also highlights random things and claims they are not being touched...

I'll keep looking for a way of properly stopping that...